libsovtoken/tests/payment_chaos_tests.rs

```diff
@@ -0,0 +1,394 @@
+extern crate libc;
+extern crate sovtoken;
+extern crate indy;                      // lib-sdk project
+
+#[macro_use] extern crate serde_derive;
+#[macro_use] extern crate serde_json;
+
+pub mod utils;
+
+use indy::ErrorCode;
+use utils::setup::{Setup, SetupConfig};
+use utils::wallet::Wallet;
+
+// ------------------------------------------------------------------------------------------------
+#[test]
+pub fn pay_without_outputs_fails() {
+
+    // ---- setup
+    sovtoken::api::sovtoken_init();
```

Comment:
This is called inside `Setup::new` and doesn't need to be called if you use the Setup function.

Reply:
Has this been resolved?

Reply:
it has not.  good catch.  I will address that now.

Reply:
latest change addresses this